src/test/java/automationpractice/com/pages/MyAccountPage.java

```diff
@@ -8,8 +8,15 @@
 
     @FindBy(className = "icon-building")
     private WebElementFacade myAddress;
+    @FindBy(id = "header_logo")
+    private WebElementFacade myStore;
 
     public void clickMyAddresses() {
         myAddress.click();
     }
+
+    public void clickMyStore() {
+        myStore.click();
+    }
+
```

Comment:
Stworzyłeś tą metodę, ale nigdzie jej nie używasz. Jeśli nigdzie jej nie używasz to ją usuń